experimental/decorators.md

```diff
@@ -0,0 +1,26 @@
+# [Decorators](https://github.com/wycats/javascript-decorators)
+
+## Decorator
+
+```js
+interface Decorator <: Node {
+    type: "Decorator";
+    expression: Expression;
+}
+```
+
+## Used
+
+```js
+extend interface MethodDefinition <: Node {
```

Comment:
This is an extension of existing `interface`, so it can't have parent specifier `<: Node`.